app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -432,7 +432,7 @@ def concern_has_file_sets?
       when ActiveFedora::Common
         curation_concern.file_sets.present?
       else
-        Hyrax.custom_queries.find_child_fileset_ids(resource: curation_concern).any?
+        Hyrax.custom_queries.find_child_file_set_ids(resource: curation_concern).any?
```

Comment:
See PR description for rationale related to renaming the `find_child_fileset*` methods to `find_child_file_set*`.